deps/wazuh_testing/wazuh_testing/scripts/qa_ctl.py

```diff
@@ -113,7 +114,8 @@ def set_environment(parameters):
     """
     if parameters.run_test:
         # Download wazuh-qa repository locally to run qa-docs tool and get the tests info
-        local_actions.download_local_wazuh_qa_repository(branch=parameters.qa_branch, path=gettempdir())
+        recursive_directory_creation(os.path.join(gettempdir(), 'qa_ctl'))
```

Comment:
Maybe this should be outside the `if` (find a good place to put it), because in the case of manual execution, this directory would not be created and would fail.

Reply:
This change has been made in the commit [8e0c857](https://github.com/wazuh/wazuh-qa/commit/8e0c8573d76c0285c5bd9a388bc07ebdfb3bccfe). The folder creation process is located now in the main function of `qa_ctl.py`. With this solution, the qa_ctl folder will be always created, either in the case of an automated execution or a manual one.